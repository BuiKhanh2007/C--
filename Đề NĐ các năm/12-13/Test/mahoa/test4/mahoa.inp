7
gkrcw
 cbds
 f zl
wixcf
sx   
