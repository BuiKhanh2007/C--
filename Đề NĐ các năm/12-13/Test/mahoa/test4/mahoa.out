nryj~
 jikz
 m Ås
~pjm
z   
